test/cluster_helper.go

```diff
@@ -1123,8 +1123,10 @@ func createDummyAerospikeCluster(
 						"proto-fd-max":     defaultProtofdmax,
 						"auto-pin":         "none",
 					},
-					"security": map[string]interface{}{},
-					"network":  getNetworkConfig(),
+					"security": map[string]interface{}{
```

Comment:
This func is used in pre-7.1.0 clusters as well. This field is not valid for them. Create another func.